ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -62,14 +62,26 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-27/temp_commit:
+  fedora-27/temp_trust:
     requires: [fedora-27/build]
     priority: 50
     job:
       class: RunPytest
```

Comment:
The suite `test_trust` needs `class: RunADTests` to be executed. Otherwise all the tests will be skipped.